tests/unit/Infrastructure/Manage/Dto/ManageEntityTest.php

```diff
@@ -104,17 +104,84 @@ public function test_diff_oidc()
         $this->assertCount(2, $diffResults['metaDataFields.redirectUrls']);
     }
 
-    public function test_is_status_publication_requested()
-    {
-        $entity = ManageEntity::fromApiResponse(json_decode(file_get_contents(__DIR__ . '/fixture/saml20_sp_requested.json'), true));
-        $entity->updateStatus(Constants::STATE_PUBLICATION_REQUESTED);
-        $this->assertEquals(Constants::STATE_PUBLICATION_REQUESTED, $entity->getStatus());
-    }
-
     public function test_is_requested_production_entity_copy()
     {
         $entity = ManageEntity::fromApiResponse(json_decode(file_get_contents(__DIR__ . '/fixture/saml20_sp_requested.json'), true));
         $this->assertTrue($entity->isRequestedProductionEntity(true));
         $this->assertFalse($entity->isRequestedProductionEntity(false));
     }
+
+    public function test_it_can_has_status_published_when_excluded_from_push()
```

Comment:
Nitpick: `it can has`

Reminds me of the good old lolcat days [icanhazcheeseburger](https://icanhas.cheezburger.com/)